### PR TITLE
[Teams] Add feature availability table to WARP pages

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/local-domains.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ pcx-content-type: how-to
  
 # Local Domain Fallback
 
+<details>
+<summary>Feature availability</summary>
+<div>
+
+| Operating Systems | [WARP mode required](/connections/connect-devices/warp#warp-client-modes) | [Teams plans](https://www.cloudflare.com/teams-pricing/) |
+| ----------------- | --------- | ---- |
+| All systems | WARP with Gateway | All plans |
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/local-domains.md:14]: @kkrum can you check that `WARP mode` is accurate?

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 2]
```diff
@@ -15,10 +15,32 @@ To manage WARP settings and device connectivity preferences, navigate to **Setti
 
 ## Admin override
 
+<details>
+<summary>Feature availability</summary>
+<div>
+
+| Operating Systems | [WARP mode required](/connections/connect-devices/warp#warp-client-modes) | [Teams plans](https://www.cloudflare.com/teams-pricing/) |
+| ----------------- | --------- | ---- |
+| macOS, Windows, Linux | WARP with Gateway | All plans |
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md]: @kkrum can you check that `WARP mode` is accurate?

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 3]
```diff
@@ -28,18 +50,51 @@ Cloudflare for Teams allows you to establish which users in your organization ca
 
 ## Captive portal detection
 
+<details>
+<summary>Feature availability</summary>
+<div>
+
+| Operating Systems | [WARP mode required](/connections/connect-devices/warp#warp-client-modes) | [Teams plans](https://www.cloudflare.com/teams-pricing/) |
+| ----------------- | --------- | ---- |
+| macOS, Windows | WARP with Gateway | All plans |
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md:59]: @kkrum can you check that `WARP mode` is accurate?

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 4]
```diff
@@ -28,18 +50,51 @@ Cloudflare for Teams allows you to establish which users in your organization ca
...
 
 ## Mode switch
 
+<details>
+<summary>Feature availability</summary>
+<div>
+
+| Operating Systems | [WARP mode required](/connections/connect-devices/warp#warp-client-modes) | [Teams plans](https://www.cloudflare.com/teams-pricing/) |
+| ----------------- | --------- | ---- |
+| macOS, Windows, Linux | WARP with Gateway | All plans |
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md:76]: @kkrum can you check that `WARP mode` is accurate?

File: products/cloudflare-one/src/content/identity/devices/require-gateway.md [hunk 5]
```diff
@@ -6,6 +6,17 @@ pcx-content-type: how-to
 
 # Require Gateway
 
+<details>
+<summary>Feature availability</summary>
+<div>
+
+| Operating Systems | [WARP mode required](/connections/connect-devices/warp#warp-client-modes) | [Teams plans](https://www.cloudflare.com/teams-pricing/) |
+| ----------------- | --------- | ---- |
+| All systems | WARP with Gateway | All plans |
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/identity/devices/require-gateway.md:15]: @kkrum can we say all systems here and fpr `Require WARP`?

File: products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/local-domains.md (hunk 1 above)
Comment (kkrum) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/local-domains.md:14]: It is correct

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md (hunk 2 above)
Comment (kkrum) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md]: This feature works with all modes

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md (hunk 3 above)
Comment (kkrum) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md:59]: This is correct

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md (hunk 4 above)
Comment (kkrum) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md:76]: This is correct

File: products/cloudflare-one/src/content/identity/devices/azure-ad.md [hunk 6]
```diff
@@ -5,6 +5,17 @@ pcx-content-type: how-to
 
 # Azure AD
 
+<details>
+<summary>Feature availability</summary>
+<div>
+
+| Operating Systems | [WARP mode required](/connections/connect-devices/warp#warp-client-modes) | Minimum WARP version required | [Teams plans](https://www.cloudflare.com/teams-pricing/) |
+| ----------------- | --------- | ---------- | ---- |
+| Windows | WARP not required | n/a | All plans |
```
Comment (kkrum) [inline products/cloudflare-one/src/content/identity/devices/azure-ad.md]: WARP With Gateway is required for this.

File: products/cloudflare-one/src/content/identity/devices/domain-joined.md [hunk 7]
```diff
@@ -6,12 +6,18 @@ title: Domain Joined
...
+| Beta | Windows | WARP with Gateway | All plans | 
 
-| Prerequisites | 
-| ------------- |
-| This check is supported on: Windows|
-| This check is not supported on: macOS, Linux, Chrome, Android, or iOS |
+</div>
+</details>
+
+The Domain Joined device posture attribute ensures that a user is a member of a specific Windows domain.
```
Comment (kkrum) [inline products/cloudflare-one/src/content/identity/devices/domain-joined.md]: Can you change it to "member of a specific Windows Active Directory domain". Need to call out Active Directory specifically as Windows has other models of domains that this does not working with.

File: products/cloudflare-one/src/content/identity/devices/require-gateway.md (hunk 5 above)
Comment (kkrum) [reply products/cloudflare-one/src/content/identity/devices/require-gateway.md:15]: Yes

**Review by kkrum — APPROVED** [review]
Looks great Alice, thanks for all your work here!

**Review by kodster28 — APPROVED** [review]

**Pull request merged**